ios/RNSScreenStackAnimator.mm

```diff
@@ -5,13 +5,14 @@
 
 // proportions to default transition duration
 static const float RNSSlideOpenTransitionDurationProportion = 1;
-static const float RNSFadeOpenTransitionDurationProportion = 0.2 / 0.35;
-static const float RNSSlideCloseTransitionDurationProportion = 0.25 / 0.35;
-static const float RNSFadeCloseTransitionDurationProportion = 0.15 / 0.35;
-static const float RNSFadeCloseDelayTransitionDurationProportion = 0.1 / 0.35;
+static const float RNSFadeOpenTransitionDurationProportion = 0.2 / 0.5;
+static const float RNSSlideCloseTransitionDurationProportion = 0.25 / 0.5;
+static const float RNSFadeCloseTransitionDurationProportion = 0.15 / 0.5;
+static const float RNSFadeCloseDelayTransitionDurationProportion = 0.1 / 0.5;
```

Comment:
Where these values come from? Do you have some kind of reference here, or are these found empirically?

Reply:
Are you talking about 0.5? I captured a transition and measured the length of the video 🙈 

I think there is no any notes about that in official documentation, but it's 500ms and many iOS developers aware about that 🙃